custom_components/waste_collection_schedule/waste_collection_schedule/source/mijnafvalwijzer_nl.py

```diff
@@ -0,0 +1,122 @@
+import re
+import datetime
+import requests
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+from waste_collection_schedule.service.ICS import ICS
+
+TITLE = "Afval Wijzer"
+DESCRIPTION = "Source for all cities regions supported in mijnafvalwijzer.nl"
+URL = "https://www.mijnafvalwijzer.nl"
+TEST_CASES = {
+    "Eindhoven1": {"postcode": "5651AN", "number": "34", "add": "A"},
+    "Eindhoven2": {"postcode": "5651AN", "number": "34"},
+    "Tilburg": {"postcode": "5014NN", "number": "174"},
+    "Meierijstad": {"postcode": "5481LR", "number": "6"},
+}
+
+ICON_MAP = {   # Optional: Dict of waste types and suitable mdi icons
+    "Restafval": "mdi:trash-can",
+    "Papier en karton": "mdi:paper-roll",
+    "Groente, Fruit en Tuinafval": "mdi:leaf",
+    "PMD": "mdi:bottle-soda-classic-outline",
+    "Plastic, Metalen en Drankkartons": "mdi:bottle-soda-classic-outline",
+    "papier": "mdi:paper-roll",
+    "GFT": "mdi:leaf",
+    "GFT ": "mdi:leaf",
+    "restafval": "mdi:trash-can",
+}
+
+#### Arguments affecting the configuration GUI ####
+
+HOW_TO_GET_ARGUMENTS_DESCRIPTION = { # Optional dictionary to describe how to get the arguments, will be shown in the GUI configuration form above the input fields, does not need to be translated in all languages
+    "en": "HOW TO GET ARGUMENTS DESCRIPTION",
+    "de": "WIE MAN DIE ARGUMENTE ERHÄLT",
+    "it": "COME OTTENERE GLI ARGOMENTI",
+}
+
+PARAM_DESCRIPTIONS = { # Optional dict to describe the arguments, will be shown in the GUI configuration below the respective input field
+    "en": {
+        "postcode": "Postcode",
+        "number": "House number",
+        "add": "Addition"
+    },
+}
```

Comment:
Remove if there is nothing to describe. (This text is shown below the input field, but your text is the same as the placeholder text already populated by the name.)

Reply:
Removed